include/generators/known_polytope_generators.h

```diff
@@ -30,7 +30,7 @@ Polytope generate_cube(const unsigned int& dim, const bool& Vpoly) {
         A.resize(2 * dim, dim);
         b.resize(2 * dim);
         for (unsigned int i = 0; i < dim; ++i) {
-            b(i) = 1.0;
+            b(i) = scale;
```

Comment:
you assign `scale` which is of type `const unsigned int` to `b(i)` which is of type `Polytope::NT`, I think `scale` should be of that type too.